webroot/results/admin/scripts/import_persons.php

```diff
@@ -8,9 +8,21 @@
 // All are done in persons_finish_unfinished_ACTION.php
 // See https://github.com/thewca/worldcubeassociation.org/issues/2286
 
+// Check if any persons with numeric ID
+$compId = mysqlEscape( $compId );
+$unfinishedPersons = dbValue("
```

Comment:
nit: By the naming convention we've been following, `unfinishedPersons` would actually be an array of unfinished persons. Instead, we'd name this `unfinishedPersonsCount`, or `numberOfUnfinishedPersons` if you prefer =)